sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -887,6 +887,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val IGNORE_NULL_FIELDS_STREAMING_SCHEMA_INFERENCE =
+    buildConf("spark.sql.streaming.schemaInference.ignoreNullFields")
```

Comment:
`ignoreNullFields` is not precise. For example

```json
{"a": 1}
{"a": null}
```

The `null` value is not ignored. I would suggest `dropFieldIfAllNull`.

Reply:
ok, I used `dropFieldIfAllNull`.